modules/installation-cluster-quickly-extend-compute-nodes.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with compute nodes on AWS Local Zones) 
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with compute nodes on AWS Wavelength Zones) 
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-cluster-quickly-extend-compute-nodes_{context}"]
+ifdef::local-zone[]
+= Install a cluster quickly in AWS Local Zones
+endif::local-zone[]
+ifdef::wavelength-zone[]
+= Install a cluster quickly in AWS Wavelength Zones
+endif::wavelength-zone[]
+
+For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to {zone-type} locations. By using this installation route, the installation program automatically creates network resources and {zone-type} subnets for each zone that you defined in your configuration file. To customize the installation, you must modify parameters in the `install-config.yaml` file before you deploy the cluster.
+
```

Comment:
Suggestion: I am looking at your published output, and I am not sure this module is required. IMO you could modify  https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone#install-creating-install-config-aws-local-zones_installing-aws-localzone to state that you modify the installation configuration file to specifiy AWS Local Zones and that the installation program creates the network resources and subnets for each zone that is specified.

You could apply the same approach for Wavelength Zones. 

If you decide to keep this, please remove references to "quickly". This is subjective and TBH there is nothing quick about deploying to either zone type.

Reply:
Hi @mtulio .Would the following title work to eliminate the "quickly" reference:

> Using the installation program to create network resources and Local Zones subnets

Reply:
FWIW: the "quickly" was based in the well-known, and entry point OCP install for AWS document[1] to do **the same** steps limited to:
- enable the zone group
- create the install-config
- create cluster

Only the first bullet is additional from original document[1]. I agree that the entire book became long, It seems the two options (full-automated/quickly install, and BYO VPC) in the same book is causing that.

> Would the following title work to eliminate the "quickly" reference:

IMO no. This suggestion is a very limited reference of what is doing, and sems not to be linked with other installation books.

I believe you could start with "Installing...", following the same standard of the BYO VPC option (and many other installer books). A few suggestions:
- Installing a cluster quickly on AWS extending worker to {zone-type}
- Installing a cluster on AWS extending worker to {zone-type}
- Installing a cluster on AWS extending nodes to {zone-type}

[1] [Installing a cluster quickly on AWS](https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-default.html)

Reply:
Thanks, Marco. On searching our customer install docs, the term "quickly" is commonly used form other platforms:

![Screenshot from 2024-02-14 09-08-55](https://github.com/openshift/openshift-docs/assets/57954076/276ed7f0-1a11-413e-be22-7050bbd27846)

Reply:
At it currently stands, "Install a cluster quickly in AWS Local Zones" is a CONCEPT modules so the gerund verb form of "Installing" is not typical. If I merge "Install a cluster quickly in AWS Local Zones" and "Modifying an installation configuration file to use AWS Wavelength Zones" together would this work? I could then create a PROCEDURE module and use "Installing" in the header·

Reply:
As it stands @dfitzmau I believe the content is technically accurate and complete. This comment came from a place of minimalism. I believe this section could be simplified, but let's take it as an item for after GA.

Reply:
Yes,I can revisit this after GA. Technical accuracy is key for now. With Oracle CLoud Infra work and release note work, the balancing act is getting tough. Thanks!